From 7e7db7225e227905acabfa2149152ece21c93e70 Mon Sep 17 00:00:00 2001
From: Steven Noonan <steven@uplinklabs.net>
Date: Sun, 8 Nov 2015 09:03:00 -0800
Subject: [PATCH] python-protobuf: don't require google-apputils

This dependency is totally superfluous for successfully building/running
python-protobuf. It's only "required" at build time and is not staged into the
install directory, but it isn't even really required for a successful build.

Signed-off-by: Steven Noonan <steven@uplinklabs.net>
---
 python/setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 2450a77..db6f497 100755
--- a/python/setup.py
+++ b/python/setup.py
@@ -160,7 +160,6 @@ if __name__ == '__main__':
         packages = [ 'google' ],
         namespace_packages = [ 'google' ],
         test_suite = 'setup.MakeTestSuite',
-        google_test_dir = "google/protobuf/internal",
         # Must list modules explicitly so that we don't install tests.
         py_modules = [
           'google.protobuf.internal.api_implementation',
@@ -189,7 +188,6 @@ if __name__ == '__main__':
           'google.protobuf.text_format'],
         cmdclass = { 'clean': clean, 'build_py': build_py },
         install_requires = ['setuptools'],
-        setup_requires = ['google-apputils'],
         ext_modules = ext_module_list,
         url = 'https://developers.google.com/protocol-buffers/',
         maintainer = maintainer_email,
-- 
2.6.2

